docs/repositories-configure/managing-branches.md

```diff
@@ -1,17 +1,38 @@
 # Managing branches
 
+{% if not extra.self_hosted %}
+Codacy automatically analyzes the default branch of your repository (typically `master` or `main` as configured on your Git provider), and also supports analyzing multiple branches.
+
+To change the default branch of your repository or enable analysis on other branches:
+
+1.  Open your repository **Settings**, tab **Branches**.
+
+    ![Managing branches](images/managing-branches.png)
+
+1.  To enable analysis for a new branch, toggle the corresponding switch in the column **Analyze**.
+
+    Enabling a new branch triggers an initial analysis of that branch and the analysis results and information for that branch will become available after the analysis is complete.
+
+1.  To change the default branch on Codacy, click the corresponding **Make default** link that appears when you hover the column **Default**.
+
+    Codacy displays information for the default branch on dashboards and the repositories list by default, and we recommend that you set the default branch on Codacy to match the default branch of your Git repository.
+
+    !!! note
+        You can only set as default branch an already enabled branch.
```

Comment:
@alerizzo After reading [your clarification](https://github.com/codacy/codacy-website/pull/286#discussion_r1000297455) I realize it's perhaps a good idea to make it more explicit that users have to wait until a branch is actually **analyzed** before they can set it as default, do you agree?

```suggestion
        You can only set as default branch an already enabled and analyzed branch.
```

Reply:
We can, although I think the API is not waiting until the end of the first analysis. Only until the enable change was done.

Reply:
Thanks for clarifying! :bow: We'll leave it as is.